src/node_allocator.hpp

```diff
@@ -42,4 +44,9 @@ struct NodeAllocator<T, Ts...> {
 	template <typename U>
```

Comment:
Should probably delete this class' copy constructor (you will then need to default the default and move constructors)

Reply:
I did it for debugging and I decided to leave it in already